layouts/partials/pixels/gtm-noscript.html

```diff
@@ -0,0 +1,11 @@
+{{ with site.Data.pixels }}
+{{ if .gtm_enabled }}
+
+<!-- Google Tag Manager (noscript) -->
+<noscript>
```

Comment:
Add the `uses-cookies` attribute here to make it GDPR compliant

Reply:
Fixed!